test/programs/method-extreme/schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "object",
+  "properties": {
+    "setTime": {
+      "parameters": [
+        {
+          "name": "d",
+          "type": "union",
+          "typeArguments": [
```

Comment:
I'm not familiar with this syntax. Can you send me some resources so I can evaluate this?